components/SendMoneyToCollectiveBtn.js

```diff
@@ -58,6 +58,9 @@ class SendMoneyToCollectiveBtn extends React.Component {
       description,
       paymentMethod: { id: paymentMethods[0].id },
       frequency: 'ONETIME',
+      customData: {
+        isBalanceTransfer: true,
```

Comment:
Seems a bit hacky to use `customData`, this is not what this field was intended to. I would rather use a dedicated GraphQL param.

Reply:
@Betree : Yes, makes sense. I've introduced the `isBalanceTransfer` as a dedicated parameter. Let me know if all looks good now. 😃